scripts/mbedtls_dev/bignum_mod.py

```diff
@@ -55,6 +55,20 @@ def result(self) -> List[str]:
 # END MERGE SLOT 4
 
 # BEGIN MERGE SLOT 5
+class BignumModAdd(bignum_common.ModOperationCommon, BignumModTarget):
+    """Test cases for bignum mpi_mod_add()."""
+    count = 0
+    symbol = "+"
+    test_function = "mpi_mod_add"
+    test_name = "mbedtls_mpi_mod_add"
+    input_style = "fixed"
```

Comment:
I'd still like to understand this.

Reply:
I'd suggest this question would be better suited for a discussion on Slack, as at this point the Python framework is being used as a "little language" to define the tests. If I had to guess I'd say "just because that's the way it was done"

Reply:
It's my bad, I didn't know that we are using the enum package and didn't want to import it.

Reply:
Anything from the [Python 3.5 standard library](https://docs.python.org/3.5/library/) is ok (except for `config.py` which should stick to 3.4 in 2.28). More recent library additions are currently not supported because much of our CI runs on 3.5.